### ISSUE TITLE
random power up spawning errors 

### ISSUE THREAD

**Issue by deathking61** [Zet0rz/nzombies#569]
Whenever I'm playing this mode the random powerup spawning works but it doesn't actually spawn anything so the console is flooded with errors (ill grabe thoes and update this issue later today.

**Comment by func-octo** [comment]
Can you post the table that is printed when you run `lua_run PrintTable(hook.GetTable())` ? Want to make sure no other addons are conflicting with random powerups.

**Comment by deathking61** [comment]
ok ill let you know when i get home tonight

**Comment by deathking61** [comment]
yes as soon as i try to get a power-up ill copy past it in here from wither
the wonder fizz machine or the random ones that spawn

**Comment by deathking61** [comment]
ok so making a map with this thing (it was a sandbox map) and as soon as i
tried to spawn it didn't have map meshes and i couldn't make one using your
mod and when i tried to save the game just crashed

**Comment by deathking61** [comment]
ok, so when the random powerup spawns i get this and it spams the console:




[ERROR]
addons/nzombies-master-workshop/gamemodes/nzombies/entities/entities/drop_powerup/shared.lua:70:
attempt to compare nil with number
  1. unknown -
addons/nzombies-master-workshop/gamemodes/nzombies/entities/entities/drop_powerup/shared.lua:70
   2. DrawModel - [C]:-1
    3. Render - lua/includes/modules/halo.lua:77
     4. fn - lua/includes/modules/halo.lua:153
      5. unknown - addons/ulib/lua/ulib/shared/hook.lua:109

**Comment by deathking61** [comment]
ok well as of now i don't think ill be playing this mode as its way to
buggy and you guys should have looked into your bugs before you released
this mode (don't get me wrong its a good thought) but its far too buggy
half the stuff doesn't work right and the other have is so glitchy that if
you go out of the area for them to pawn you will lag so hard the game will
freeze up. so I'm done I cannot play a mode that was half-assed im sorry
but if you need me to look into other bugs i can but other than that i
cannot play this.

**Comment by bozoweed** [comment]
@deathking61 i used this mod for personal use and it's work fine , yes they got some litlle bug but we can played on it or fix the ia's bug by remake the meshes of the map for better ia way ( they have tuto for that on github) , but maybe your source is buged it's not impossible try to download the update and install it correctly and look if it's better if it's not i will go to seen for fix the bug, but please look what you have beetwin your hand and say thanks for the work and leak that on github.
